### ISSUE TITLE
SideTone

### ISSUE THREAD

**Issue by remorse** [dfanara/Arctis-on-Linux#1]
Hi! I bought the Arctis Nova 3, but it comes with SideTone turned on and no way to turn it off unless you're on a Windows computer (and I'm on an M3 Mac). How did you figure out what commands to send to the device? If possible, I'd like to just write something that turns SideTone off without doing anything else...

Thanks!
Ricky

**Comment by dfanara** [comment]
Hey Ricky,

Thanks for reaching out. 

It's been a while since I worked on this, but if I recall correctly my general process was this:

1. Install the Steelseries GG software on linux using Wine
2. Install Wireshark and setup USB packet capturing on linux to capture the control traffic from Steelseries GG to the Arctis base station. 
3. I would filter the wireshark traffic down based on the vendor code for the headset. I don't remember exactly how I found that, but I believe I used `lsusb` or a similar command. 
4. I would trigger one function in the software and watch the packet, trying to decode the individual parameters. As far as I remember, they were pretty straight forward to reverse engineer, I don't recall any encrypted values or anything like that.

Hope that helps!

Daniel
